types/shared.d.ts

```diff
@@ -355,33 +357,24 @@ export interface IStripeConnectInitParams {
   publishableKey: string;
 
   /**
-   * The client secret for the connected account.
+   * Function that fetches client secret
+   * @returns A promise that resolves with a new client secret.
    */
-  clientSecret: string;
+  fetchClientSecret: () => Promise<string>;
 
   /**
    * Appearance options for the Connect instance.
    * @see https://stripe.com/docs/connect/customize-connect-embedded-components
    */
   appearance?: AppearanceOptions;
 
-  /**
-   * Callback function that returns a new client secret. Used to support long running sessions and called on account session expiry.
-   */
-  refreshClientSecret?: () => Promise<string>;
-
   /**
    * The locale to use for the Connect instance.
    */
   locale?: string;
 }
 
 export interface StripeConnectWrapper {
-  /**
```

Comment:
Why remove this part?

Reply:
External users shouldn't be accessing the StripeConnectWrapper directly so I undocumented it, should I remove it from types all together?

Reply:
Ahh yes we should! If it is needed for our code we can still have it in another TS file (i.e. inline), just without exporting it.